examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -2172,6 +2172,22 @@
               "source": "server",
               "isIncoming": 0,
               "isEnabled": 1
+            },
+            {
+              "name": "SetTCAcknowledgements",
+              "code": 6,
+              "mfgCode": null,
+              "source": "client",
+              "isIncoming": 1,
+              "isEnabled": 1
+            },
+            {
+              "name": "SetTCAcknowledgementsResponse",
+              "code": 7,
+              "mfgCode": null,
+              "source": "server",
+              "isIncoming": 0,
+              "isEnabled": 1
```

Comment:
Please do not T&Cs to all-clusters-app, but rather create a new ESF sample, as otherwise we cannot guarantee how these invariants will be maintained forever, and this will also mean that all-cluster-app may get setup in the future with mandatory terms, which would break many development usecases.

This was already mentionned to @koosc in the last ESF call.